practical-react-unit-testing-with-jest-mocks-and-spies/Exercise3/solution/src/Select.test.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
+ */
+//auto mocking toSelectOptions
```

Comment:
I’m having a hard time spotting the difference between the problem and solution… can you point it out to me or does the problem need some code removed?

I think including the exercise prompt in the file would make this more clear. Example: https://github.com/bitovi/trainings/blob/b0e9caa09a13d00e8d76ae43da2ec1bbdae87560/typing-react-state/Exercise3/problem/src/App.tsx#L3-L6